x-pack/dockerlogbeat/magefile.go

```diff
@@ -77,10 +77,10 @@ func createContainer(ctx context.Context, cli *client.Client) error {
 	}
 
 	// change back to the root beats dir so we can send the proper build context to docker
-	err = os.Chdir("../..")
-	if err != nil {
-		return errors.Wrap(err, "error changing directory")
-	}
+	// err = os.Chdir("../..")
```

Comment:
If it's not needed then please remove all the commented out code. BTW using all of the Beats repo as context for the build without having any .dockerignore files would definitely take some time and disk space in some environments where the whole directory must be copied recursively to the docker VM. The repo dir will have all of the binary packages of other beats that were previously built.

Reply:
Yah, that should be gone now.